pmda/test/test_rms.py

```diff
@@ -0,0 +1,62 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# PMDA
+# Copyright (c) 2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+from __future__ import absolute_import, division, print_function
+
+import pytest
+from numpy.testing import (assert_almost_equal, assert_array_equal,
+                           assert_array_almost_equal)
+from distutils.version import LooseVersion
+
+import MDAnalysis
+from MDAnalysisTests.datafiles import (PSF, DCD)
+
+from pmda import rms
+
+# remove when MDAnalysis 0.18.0 is available
+xfail_mda_issue1819 = pytest.mark.xfail(
+    LooseVersion(MDAnalysis.__version__) <= LooseVersion("0.17.0"),
+    reason="Fails because of MDAnalysis issue #1819")
+
+
+class TestRMSD(object):
+    @pytest.fixture()
+    def universe(self):
+        return MDAnalysis.Universe(PSF, DCD)
+
+    @pytest.fixture()
+    def correct_values(self):
+        return [[0, 1.0, 0], [49, 50.0, 4.68953]]
+
+    @pytest.fixture()
+    def correct_values_frame_5(self):
+        return [[5, 6.0, 0.91544906]]
+
+    def test_rmsd(self, universe):
+        ca = universe.select_atoms("name CA")
+        universe.trajectory.rewind()
+        RMSD = rms.RMSD(ca, ca).run(n_jobs=2)
+        assert_array_equal(RMSD.rmsd.shape, (universe.trajectory.n_frames, 3))
+
+    @xfail_mda_issue1819
```

Comment:
Locally the xfail works (eg when I change 0.17.0 to 0.19.0) but not on travis. Not sure why.